service/src/test/java/bio/terra/catalog/service/JsonValidationServiceTest.java

```diff
@@ -0,0 +1,32 @@
+package bio.terra.catalog.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import bio.terra.catalog.config.BeanConfig;
+import bio.terra.catalog.config.SchemaConfiguration;
+import bio.terra.common.exception.BadRequestException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.junit.jupiter.api.Test;
+
+public class JsonValidationServiceTest {
+  private static final String BASE_PATH = "src/test/resources/schemas/test.schema.json";
+
+  private static final ObjectMapper objectMapper = new BeanConfig().objectMapper();
+
+  @Test
+  void testValidateInvalidMetadata() {
```

Comment:
Can you add another test that verifies that an invalid schema will throw an error during object construction? Once you've tested that, you should have 100% coverage of the new service code. You can look at the coverage of the test by running it in IntelliJ.